new

joshwhit99 opened the pull request:
new

joshwhit99 committed:
Update details.html

joshwhit99 committed:
Update controllers.js

akasht committed:
Adding Cloud Web Service for Curae Analytics

ez9115 commented:
Josh , are we in sync with entire team about directions. i am kind of lost
at this point.
What should be a next step ?

On Thu, Nov 5, 2015 at 6:54 PM, joshwhit99 notifications@github.com wrote:

> ## new
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/joshwhit99/CuraeApp/pull/1
> Commit Summary
> - Update details.html
> - Update controllers.js
> 
> File Changes
> - _M_ app/js/controllers.js
>   https://github.com/joshwhit99/CuraeApp/pull/1/files#diff-0 (4)
> - _M_ app/partials/details.html
>   https://github.com/joshwhit99/CuraeApp/pull/1/files#diff-1 (4)
> 
> Patch Links:
> - https://github.com/joshwhit99/CuraeApp/pull/1.patch
> - https://github.com/joshwhit99/CuraeApp/pull/1.diff
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/joshwhit99/CuraeApp/pull/1.

## 

Thank you ,
Elena Zlatopolsky
404-510-1740

joshwhit99 committed:
Create bootstrap.css

joshwhit99 committed:
Create bootstrap.css.map

joshwhit99 committed:
Create bootstrap.min

joshwhit99 committed:
Create bootstrap-theme.css

joshwhit99 committed:
Create bootstrap-theme.css.map

joshwhit99 committed:
Create bootstrap-theme.min

joshwhit99 committed:
Create main.css

joshwhit99 committed:
Update app.js

joshwhit99 committed:
Update controllers.js

joshwhit99 committed:
Update data.json

joshwhit99 committed:
Create data66.json

joshwhit99 committed:
Create angular-chart.css

joshwhit99 committed:
Create angular-chart.js

joshwhit99 committed:
Create angular-fusioncharts.min.js

joshwhit99 committed:
Create bootstrap.js

joshwhit99 committed:
Create bootstrap.min

joshwhit99 committed:
Rename bootstrap.min to bootstrap.min.js

joshwhit99 committed:
Create Chart.js

joshwhit99 committed:
Create fusioncharts.charts.js

joshwhit99 committed:
Create fusioncharts.gantt.js

joshwhit99 committed:
Create fusioncharts.js

joshwhit99 committed:
Create fusioncharts.maps.js

joshwhit99 committed:
Create fusioncharts.powercharts.js

joshwhit99 committed:
Create fusioncharts.widgets.js

joshwhit99 committed:
Create jquery-1.11.2.js

joshwhit99 committed:
Create fusioncharts.theme.carbon

joshwhit99 committed:
Create fusioncharts.theme.fint

joshwhit99 committed:
Create fusioncharts.theme.ocean

joshwhit99 committed:
Create fusioncharts.theme.zune.js

joshwhit99 committed:
Rename fusioncharts.theme.ocean to fusioncharts.theme.ocean.js

joshwhit99 committed:
Rename fusioncharts.theme.fint to fusioncharts.theme.fint.js

joshwhit99 committed:
Rename fusioncharts.theme.carbon to fusioncharts.theme.carbon.js

joshwhit99 committed:
Rename bootstrap.min to bootstrap.min.css

joshwhit99 committed:
Rename bootstrap-theme.min to bootstrap-theme.min.css

joshwhit99 committed:
Update details.html

joshwhit99 committed:
Update list.html

joshwhit99 committed:
Create nodiabetes.html

joshwhit99 committed:
Create reports.html

joshwhit99 committed:
Update index.html

joshwhit99 committed:
I took out the left and right button.

joshwhit99 committed:
i added the '&_sort:desc&_count=1' to -

the end of some of the getjson pulls so that they come in order and i cleaned up the code by taking out old comments.

joshwhit99 committed:
Create technology_used.html

joshwhit99 committed:
Update list.html

joshwhit99 committed:
Update controllers.js

joshwhit99 committed:
Update app.js

akasht committed:
Delete old web solution

akasht committed:
Added Curae Analytics Solution

joshwhit99 committed:
Update details.html

joshwhit99 committed:
Update details.html

dmacanaspie committed:
Update Index.html

Changed Title

dmacanaspie committed:
Update style.css

Added style for charts page

dmacanaspie committed:
Update details.html

Changed wording

dmacanaspie committed:
Update list.html

Updated working in header class="raw-fluid"

dmacanaspie committed:
Update reports.html

Changed "Patient Data" and "Population Data" formatting to text-align:center and stand-alone row.

dmacanaspie committed:
Update controller.js

Replaced controller.js with copy from master app/js/controller.js

dmacanaspie committed:
Create technology_used.html

Copied file from master/app/partials

dmacanaspie committed:
Update details.html

Added "Address:" before patient address

joshwhit99 committed:
Update data66.json

dmacanaspie committed:
Update style.css

Replaces existing style.css with colors and fonts from the main pages.

dmacanaspie committed:
Update nodiabetes.html

Updated formatting for charts to make consistent with reports.html

dmacanaspie committed:
Update reports.html

Updated header text and "back to search" button

dmacanaspie committed:
Update nodiabetes.html

dmacanaspie committed:
update app.js

Replace app.js with master/app/js/app.js

dmacanaspie committed:
Rejected commit: Default

dmacanaspie committed:
Update README.md

Created base for application CureaApp

dmacanaspie committed:
Update README.md

Fix formatting

dmacanaspie committed:
Update technology_used.html

Updated href for style ./css/*.css to ../css/bootstrap.css etc.  Invalid directory reference was changed for "Return to Main Page" button as well.

dmacanaspie committed:
Update list.html

Changed href for Technology Used button to ./technology_used.html.  Routing method #/technology_used was not working on the website even though it was working locally.

dmacanaspie committed:
Update nodiabetes.html

Changed button href="#/index.html" to "../index.html" Makes the href reference explicit rather than through route parameters in AngularJS.
